guides/app-guides/borrow-ageur.md

```diff
@@ -0,0 +1,79 @@
+---
```

Comment:
Need to update the README of App guides to link the new borrow and leverage guides